plugins/modules/aci_fabric_node.py

```diff
@@ -222,6 +234,8 @@ def main():
         node_id=dict(type="int"),  # Not required for querying all objects
         pod_id=dict(type="int"),
         role=dict(type="str", choices=["leaf", "spine", "unspecified"], aliases=["role_name"]),
+        node_type=dict(type="str", choices=["tier-2", "virtual", "unspecified"]),
```

Comment:
think we should leverage the constant here also
```suggestion
        node_type=dict(type="str", choices=list(NODE_TYPE_MAPPING.keys())),
```